packages/concerto-vocabulary/types/lib/vocabularymanager.d.ts

```diff
@@ -96,10 +97,21 @@ declare class VocabularyManager {
      * @returns {string} the term or null if it does not exist
      */
     getTerm(namespace: string, locale: string, declarationName: string, propertyName?: string): string;
+    /**
+     * Creates a DecoractorCommandSet with @Term decorators
+     * to decorate all model elements based on the vocabulary for a locale.
+     * Pass the return value to the DecoratorManager.decorateModel to apply
+     * the decorators to a ModelManager.
+     * @param {ModelManager} modelManager - the Model Manager
+     * @param {string} locale the BCP-47 locale identifier
+     * @returns {*} the decorator command set used to decorate the model.
```

Comment:
We can do a better job here with the type? We have the CTO model for a Decorator command set and can generate a TypeScript file from it?